core/envelope-sim/src/testFixtures/java/fr/sncf/osrd/envelope_sim/SimpleRollingStock.java

```diff
@@ -90,6 +90,47 @@ public double getRollingResistanceDeriv(double speed) {
         return B + 2 * C * speed;
     }
 
+    // TODO: implement these methods?
+    @Override
+    public double getTractionCutOff() {
+        return 0;
```

Comment:
(same for all) how about:
```suggestion
        throw new UnsupportedOperationException("To be implemented");
```

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/10014/commits/7eccec26e618803b194df654161dfc8dd0b130ed
However this will probably change when I start adding unit tests.